libmscore/repeatlist.cpp

```diff
@@ -389,11 +389,11 @@ void RepeatList::collectRepeatListElements()
                         remainder = swap;
                     }
                     // Cross-section of the repeatList
-                    std::list<int> endings = remainder->endings().toStdList();
+                    std::list<int> endings(remainder->endings().begin(), remainder->endings().end());
```

Comment:
@jeetee please check this one here

Reply:
Agreed, master only as per https://doc.qt.io/qt-5/qlist.html#fromStdList

Reply:
And more importantly, according to https://www.cplusplus.com/reference/list/list/list/ variant 3